cmd/tsidp/tsidp.go

```diff
@@ -72,6 +72,7 @@ var (
 	flagFunnel             = flag.Bool("funnel", false, "use Tailscale Funnel to make tsidp available on the public internet")
 	flagHostname           = flag.String("hostname", "idp", "tsnet hostname to use instead of idp")
 	flagDir                = flag.String("dir", "", "tsnet state directory; a default one will be created if not provided")
+	flagControlURL         = flag.String("login-server", ipn.DefaultControlURL, "base URL of control server")
```

Comment:
I'd just remove this default. And then document it as "optional alternate control server base URL. If empty, the default Tailscale control plane is used."